co-log-core/co-log-core.cabal

```diff
@@ -47,6 +47,6 @@ library
 test-suite doctest
   build-depends: base, doctest ^>= 0.16.0
   default-language: Haskell2010
-  hs-source-dirs: tests
-  main-is: doctest.hs
+  hs-source-dirs: test
+  main-is: Doctests.hs
   type: exitcode-stdio-1.0
```

Comment:
Could you, please, format `test-suite` with alignment as it was previously in your PR? :slightly_smiling_face: